members_2023.html

```diff
@@ -0,0 +1,765 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Members-2021</title>
```

Comment:
Please change the Title of the website accordingly

![image](https://github.com/WiOSc/Wios-New-Web/assets/107696874/df8c1af9-3765-406a-9bbb-b85bc78bf6a7)